test/utils/windowUtilsTests.ts

```diff
@@ -1,8 +1,52 @@
 ///<reference path="../testReference.ts" />
 
 describe("Utils.Window", () => {
-  describe("deprecated()", () => {
+  describe("warn()", () => {
```

Comment:
Appears to work correctly for `describe()`s:

<img width="466" alt="screen shot 2015-10-22 at 4 15 57 pm" src="https://cloud.githubusercontent.com/assets/1440449/10680829/33aaf512-78d8-11e5-8d38-832ec3fb66ee.png">